osgar/drivers/gps.py

```diff
@@ -43,12 +45,75 @@ def parse_line(line):
         coord = [str2ms(s[4]), str2ms(s[2])]
         return coord
 
+    @staticmethod
+    def parse_bin(data):
```

Comment:
Is there a specific reason why have `@staticmethod`s compared to plain old functions in module scope like `checksum`? I find the slightly less intuitive since I seem to always miss the decorator at the beginning of the review.